content/telegraf/v1.22/configure_plugins/external_plugins/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Integrate with external plugins
+description:
+menu:
+  telegraf_1_22:
+
+     name: External plugins
+     weight: 50
+     parent: Configure plugins
+---
+
+[External plugins](/EXTERNAL_PLUGINS.md) are external programs that are built outside
+of Telegraf that can run through an `execd` plugin. These external plugins allow for
+more flexibility compared to internal Telegraf plugins. Benefits to using external plugins include:
+- Access to libraries not written in Go
+- Using licensed software (not available to open source community)
+- Including large dependencies that would otherwise bloat Telegraf
+- Using your external plugin immediately without waiting for the Telegraf team to publish
+- Easily convert plugins between internal and external using the [shim](/telegraf/latest/external_plugins/shim/)
```

Comment:
broken link..did we drop shim? (did a search and can't find)

let's run a link check on this branch being so much has moved around
& do a search and replace `/telegraf/latest` with `/{{< latest "telegraf" >}}`